src/main/resources/graphql/journalpost.graphql

```diff
@@ -0,0 +1,52 @@
+query {
```

Comment:
Trengs alle disse verdiene?

Reply:
Uklart. Tok de med i tilfelle. Hvis vi skal opprette saker i våres nye sakssystem så kanskje? Validere at journalposten er god nok for å kalles "mottatt"? Vi får se. Lett å fjerne.

Reply:
:+1: